hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -120,7 +120,9 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
     OMMetadataManager metadataManager = ozoneManager.getMetadataManager();
 
-    BucketInfo bucketInfo = getBucketInfoFromRequest();
+    CreateBucketRequest createBucketRequest = getOmRequest()
```

Comment:
Any reason for removing method, and adding that logic here?

Reply:
Needed the createBucketRequest for logging in case of replay transaction. Hence, moved it from the function to avoid duplication.